Buildings/Experimental/DHC/CentralPlants/BaseClasses/PartialPlant.mo

```diff
@@ -129,6 +120,16 @@ partial model PartialPlant
     annotation (
       Placement(transformation(extent={{300,100},{340,140}}),
         iconTransformation(extent={{300,80},{380,160}})));
+protected
+  final parameter Boolean connector_hea=typ <> Buildings.Experimental.DHC.Types.DistrictSystemType.Cooling and
```

Comment:
change the name from `connector_*` to `have_heatingPort`, `have_coolingPort`, and `have_ambientPort`. It's more straightforward.

Reply:
Thanks for pointing this out! It has been changed accordingly.